src/encoder.rs

```diff
@@ -299,6 +299,7 @@ pub struct FrameState<T: Pixel> {
   pub input: Arc<Frame<T>>,
   pub input_hres: Plane<T>, // half-resolution version of input luma
   pub input_qres: Plane<T>, // quarter-resolution version of input luma
+  pub activity_mask: Plane<T>,
```

Comment:
On IRC, I think you said you wanted activity to be 16-bit always. So this should be a Plane<u16>.

Other than that lgtm.

Reply:
I've not pushed those changes yet 😅. I am looking at adding a function to retrieve the activity of a tile from this mask in TileStateMut. After that, I plan to add a unit test as discussed.

Reply:
Great, thanks!